accessible/clipboard.service.js

```diff
@@ -0,0 +1,98 @@
+var blocklyApp = blocklyApp || {};
```

Comment:
Presumably, in the instructions to developers, they will be told "load file X first", where file X contains this line. If you make that clear, I think it should be sufficient. It seems odd to guard against this but not to guard against missing files, stuff included twice, etc.

Happy to revisit this if there's prior art somewhere showing an alternative standard practice.